nginx.conf.template

```diff
@@ -44,8 +44,15 @@ http {
   # can run a BDB cluster.
   server {
     lua_code_cache on;
-    listen BIGCHAINDB_FRONTEND_PORT;
+    listen BIGCHAINDB_FRONTEND_PORT ssl;
     server_name "FRONTEND_DNS_NAME";
+
+    ssl_certificate        /usr/local/openresty/nginx/conf/ssl/cert.pem;
+    ssl_certificate_key    /usr/local/openresty/nginx/conf/ssl/cert.key;
+    ssl_protocols          TLSv1 TLSv1.1 TLSv1.2;
```

Comment:
Those are the default values in nginx. I made it explicit so that we are aware of it.
From [nginx docs](http://nginx.org/en/docs/http/configuring_https_servers.html):

> The directives ssl_protocols and ssl_ciphers can be used to limit connections to include only the strong versions and ciphers of SSL/TLS. By default nginx uses “ssl_protocols TLSv1 TLSv1.1 TLSv1.2” and “ssl_ciphers HIGH:!aNULL:!MD5”, so configuring them explicitly is generally not needed. Note that default values of these directives were changed several times.